src/nbl/ui/CWindowWin32.cpp

```diff
@@ -395,8 +406,9 @@ namespace ui
 				auto* inputChannel = window->getMouseEventChannel(device);
 				RAWMOUSE rawMouse = rawInput.data.mouse;
  
-				if ((rawMouse.usFlags & MOUSE_MOVE_RELATIVE) == MOUSE_MOVE_RELATIVE && (rawMouse.lLastX != 0 || rawMouse.lLastY != 0))
+				if ((rawMouse.usFlags & MOUSE_MOVE_RELATIVE) == MOUSE_MOVE_RELATIVE)
 				{
+					assert(rawMouse.lLastX != 0 || rawMouse.lLastY != 0);
```

Comment:
&&

Reply:
otherwise you assert on pure horizontal or pure vertical moves

Reply:
Can't one of them be 0?

Reply:
Like i need at least one coord to be not zero

Reply:
sorry I'm dumb... I forgot its `assert(true)` (I got it mixed up with BREAK_IF)